compiler/src/dotty/tools/dotc/typer/Inliner.scala

```diff
@@ -53,23 +53,13 @@ object Inliner {
       // This is quite tricky, as such types can appear anywhere, including as parts
       // of types of other things. For the moment we do nothing and complain
       // at the implicit expansion site if there's a reference to an inaccessible type.
-      override def transform(tree: Tree)(implicit ctx: Context): Tree = tree match {
-        case tree: Assign =>
-          transform(tree.lhs) match {
-            case lhs1: RefTree =>
-              lhs1.name match {
-                case InlineAccessorName(name) =>
-                  cpy.Apply(tree)(useSetter(lhs1), transform(tree.rhs) :: Nil)
-                case _ =>
-                  super.transform(tree)
-              }
-            case _ =>
-              super.transform(tree)
-          }
-        case _ =>
-          super.transform(accessorIfNeeded(tree))
-      }
-
+      override def transform(tree: Tree)(implicit ctx: Context): Tree =
```

Comment:
(I have no useful comment, but I will say that I don't understand Inliner well enough yet to really judge this change.)